src/main/webapp/dashboard.js

```diff
@@ -0,0 +1,99 @@
+// ELEMENTS_FOR_ABOUT_ME_SECTION
+/**
+ * Elements from dashboard.html.
+ * @type {HTMLElement}
+ */
+const ABOUT_ME_CARD_DIV = document.getElementById('about-me-card-body');
+const USER_BIO = document.getElementById('user-bio');
+const USER_IMAGE = document.getElementById('user-image');
+const USER_NAME_AND_LOCATION = document.getElementById('user-name-location');
+const USER_TAG_ROW = document.getElementById('user-tag-row');
+const USER_GITHUB_BUTTON = document.getElementById('user-github');
+
+/**
+ * Fake information for the hardcoded stage.
+ */
+const dummyName = 'Dior';
+const dummyLocation = 'Sunnyvale CA.';
+const dummyBio = 'The cutest pug in The tech industry. Lorem ipsum dolor sit ' +
+  'amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut' +
+  ' labore et dolore magna aliqua.';
+const dummyImg = 'images/dior.jpg';
+const dummyTags = ['Weekly', '30 mins', 'Kubernetes', 'Open Sesame',
+  'Documentation',
+];
+const dummyGitSrc = '#';
```

Comment:
You may want to store this data in the same format that you expect to get from the github API (I assume it would be a list of json objects but you should double-check). That way you'll be able to write the logic to iterate over a list of users before trying to integrate the github API with this code.

Reply:
will do!